app/v2/models/responses/hateoas/HateoasMovementUpdateResponse.scala

```diff
@@ -20,14 +20,15 @@ import play.api.libs.json.JsObject
 import play.api.libs.json.Json
 import v2.models.MessageId
 import v2.models.MovementId
+import v2.models.MovementType
 
-object HateoasDepartureUpdateMovementResponse extends HateoasResponse {
+object HateoasMovementUpdateResponse extends HateoasResponse {
 
-  def apply(departureId: MovementId, messageId: MessageId): JsObject =
+  def apply(departureId: MovementId, messageId: MessageId, movementType: MovementType): JsObject =
     Json.obj(
       "_links" -> Json.obj(
-        "self"      -> Json.obj("href" -> messageUri(departureId, messageId)),
-        "departure" -> Json.obj("href" -> departureUri(departureId))
+        "self"                          -> Json.obj("href" -> getMessageUri(departureId, messageId, movementType)),
+        s"${movementType.movementType}" -> Json.obj("href" -> getMovementUri(departureId, movementType))
```

Comment:
This can just be `movementType.movementType` -- no need to wrap it in an s-interpolated string if you're just using that string.